pyproject.toml

```diff
@@ -29,7 +29,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.1"
```

Comment:
this is a pre-commit requirement. it's possible to avoid bumping this version, by instead specifying that pre-commit is installed as a system dependency, instead of using Poetry. do you have a preference?

Reply:
It's fine to have it as project dependency, as the version diff is not really soo huge :)